src/Symfony/Component/Translation/Tests/Dumper/JsonFileDumperTest.php

```diff
@@ -30,7 +30,19 @@ public function testDump()
         $dumper->dump($catalogue, array('path' => $tempDir));
 
         $this->assertEquals(file_get_contents(__DIR__.'/../fixtures/resources.json'), file_get_contents($tempDir.'/messages.en.json'));
+        unlink($tempDir.'/messages.en.json');
```

Comment:
this won't be unlinked in case of assertion failure. You should do it on teardown